eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -34,6 +38,24 @@ foreach ($res in $responses)
   }
   $branch = $res.ref
   $branchName = $branch.Replace("refs/heads/","")
+  $branchName -match $CentralPRRegex | Out-Null
+  $pullRequestNumber = $Matches["PrNumber"]
+  if (!$pullRequestNumber) {
+    LogWarning "Did not fetch any pr number from the branch. Please double check $branchName"
+    continue
+  }
+  try {
+    $centralPR = Get-GitHubPullRequest -RepoId $CentralRepoId -PullRequestNumber $pullRequestNumber -AuthToken $AuthToken
+    if ($centralPR.state -ne "closed") {
```

Comment:
We probably should handle merged state as well.

Reply:
I think merged state is `closed` in github. So handing `closed` is enough.